File: SDKs/amazonaws.com/wellarchitected/2020-03-31/python/src/sdk/models/shared/getlensreviewreportoutput.py
from dataclasses import dataclass, field
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils
from . import *


@dataclass_json
@dataclass
class GetLensReviewReportOutput:
    r"""GetLensReviewReportOutput
    Output of a get lens review report call.
    """
    
    lens_review_report: Optional[LensReviewReport] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('LensReviewReport') }})
    milestone_number: Optional[int] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('MilestoneNumber') }})
    workload_id: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('WorkloadId') }})
    

File: SDKs/amazonaws.com/wellarchitected/2020-03-31/typescript/src/sdk/models/shared/getlensreviewreportoutput.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { LensReviewReport } from "./lensreviewreport";



// GetLensReviewReportOutput
/** 
 * Output of a get lens review report call.
**/
export class GetLensReviewReportOutput extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=LensReviewReport" })
  lensReviewReport?: LensReviewReport;

  @SpeakeasyMetadata({ data: "json, name=MilestoneNumber" })
  milestoneNumber?: number;

  @SpeakeasyMetadata({ data: "json, name=WorkloadId" })
  workloadId?: string;
}
